src/vue/backend/src/templates/importHeader.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+    <idx-tab-container
+        customClass="import-header"
+        @activeTab="switchTabs"
+        :activeTab="activeTab"
+        :tabs="['Unimported', 'Imported']"
+    >
+        <idx-block className="import-header__description">{{ description }}</idx-block>
+        <idx-block className="import-header__actions-bar">
+            <idx-block className="import-header__select-all" @click="$emit('selectAll', selected)">{{ selected ? 'Select All' : 'Deselect All' }}</idx-block>
+            <idx-button customClass="import-header__action" @click="$emit('bulkAction', action)">{{ action }} Selected</idx-button>
+        </idx-block>
+        <router-view></router-view>
+    </idx-tab-container>
+</template>
+<script>
+export default {
+    name: 'import-header',
+    data () {
+        return {
+            activeTab: 0
+        }
+    },
+    props: {
+        selected: {
+            type: Boolean,
+            default: false
+        },
+        action: {
+            type: String,
+            default: 'Import'
+        },
+        description: {
+            type: String,
+            default: ''
+        }
+    },
+    methods: {
+        switchTabs (e) {
+            this.activeTab = e
+            /* Router change here, once we have router set up we can switch it. For now emitting and event */
+            this.$emit('switchTabs', this.activeTab)
```

Comment:
Event names should be kebab-case per Vue docs: https://vuejs.org/v2/guide/components-custom-events.html#Event-Names